Module not found: Error: Default condition should be last one

my vue2 project got a webpack complie error after import markdown-it-math (version:5.2.0).

```
 error  in ./src/components/index.vue?vue&type=script&lang=js

Module not found: Error: Default condition should be last one

ERROR in ./src/components/index.vue?vue&type=script&lang=js (./node_modules/babel-loader/lib/index.js??clonedRuleSet-40.use[0]!./node_modules/@vue/vue-loader-v15/lib/index.js??vue-loader-options!./src/components/index.vue?vue&type=script&lang=js) 9:0-46
Module not found: Error: Default condition should be last one
 @ ./src/components/index.vue?vue&type=script&lang=js 1:0-198 1:214-217 1:219-414 1:219-414
 @ ./src/components/index.vue 2:0-56 3:0-51 3:0-51 9:2-8
 @ ./src/index.js 1:0-42 5:0-16 6:28-36 9:17-25 13:15-23
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-40.use[0]!./node_modules/@vue/vue-loader-v15/lib/index.js??vue-loader-options!./examples/components/HelloWorld.vue?vue&type=script&lang=js 1:0-39 5:4-12
 @ ./examples/components/HelloWorld.vue?vue&type=script&lang=js 1:0-203 1:219-222 1:224-424 1:224-424
 @ ./examples/components/HelloWorld.vue 2:0-61 3:0-56 3:0-56 10:2-8
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-40.use[0]!./node_modules/@vue/vue-loader-v15/lib/index.js??vue-loader-options!./examples/App.vue?vue&type=script&lang=js 1:0-53 5:4-14
 @ ./examples/App.vue?vue&type=script&lang=js 1:0-190 1:206-209 1:211-398 1:211-398
 @ ./examples/App.vue 2:0-54 3:0-49 3:0-49 10:2-8
 @ ./examples/main.js 2:0-28 6:17-20

webpack compiled with 1 error
```

--- a/package.json
+++ b/package.json
@@ -7,16 +7,16 @@
   "types": "types/index.js",
   "exports": {
     ".": {
-      "default": "./index.js",
-      "types": "./types/index.d.ts"
+      "types": "./types/index.d.ts",
+      "default": "./index.js"
     },
     "./no-default-renderer": {
-      "default": "./no-default-renderer.js",
-      "types": "./types/no-default-renderer.d.ts"
+      "types": "./types/no-default-renderer.d.ts",
+      "default": "./no-default-renderer.js"
     },
     "./temml": {
-      "default": "./temml.js",
-      "types": "./types/temml.d.ts"
+      "types": "./types/temml.d.ts",
+      "default": "./temml.js"
     }
   },
   "scripts": {

